iMethod.java

```diff
@@ -0,0 +1,65 @@
+package CalculaitorMain;
+
+public abstract class iMethod implements SUM,DIV,MULT,MIN {
+
+    abstract void summa();
```

Comment:
you can write an implementation within the method ex:
int summa() {
//from Sum interface
return summa(int num1, int num2);
}